packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -173,10 +173,10 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
         _removeAddresses(_members);
 
         // Check if the new address list has become shorter than the current minimum number of approvals required.
-        uint256 newAddresslistLength = addresslistLength();
-        uint256 minApprovals_ = multisigSettings.minApprovals;
-        if (newAddresslistLength < minApprovals_) {
-            revert MinApprovalsOutOfBounds({limit: newAddresslistLength, actual: minApprovals_});
+        uint16 newAddresslistLength16 = addresslistLength().toUint16();
```

Comment:
I'd not transform it to uint16. If newAddresslistLength is 67,000 , and you convert it to uint16, its value will change which is incorrect.

Reply:
Fixed in 2e57532538654d7e3d0ed481b451c0f0cea759c3

Reply:
Fixed in e434fe66a228351023cfaf9cbde8b8102695d63f

Reply:
After a discussion with @novaknole (https://discord.com/channels/672466989217873929/1063135237699289150/1063503000632569936), we agreed to convert both, `minApprovals` and `addresslistLength`, to `uint32`.